File: models/Complement.java
package models;

public class Complement {
    private int id;
    private String nome;
    private double prix;
    private String image;
    private boolean archive = false;

    public Complement(int id, String nome, double prix, String image) {
        this.id = id;
        this.nome = nome;
        this.prix = prix;
        this.image = image;
    }

    public double getPrix() { return prix; }
    public String getNom() { return nom; }
    public boolean isArchive() { return archive; }

    public void archiver() { this.archive = true; }
}

File: Models/Complement.cs
using System.ComponentModel.DataAnnotations;

namespace BrasilBurger.Models
{
    public class Complement
    {
        public int Id { get; set; }

        [Required]
        public required string Nom { get; set; }

        [Required]
        [Range(0.01, double.MaxValue)]
        public decimal Prix { get; set; }

        public required string Image { get; set; }

        public bool Archive { get; set; } = false;
    }
}